### PR TITLE
Add the ability to fetch experiment config flags from the article end…

### PR DESCRIPTION
**Pull request by polyakovn**
…point. Thank-js (and in the future, swg-js) will use this method to fetch experiment flags in their runtimes, and populate the flags in experiments.js. For reference, cl/424628526 populates the experiment configuration returned by the article endpoint.

### CODE REVIEW

File: src/runtime/entitlements-manager.js [hunk 1]
```diff
@@ -456,6 +461,23 @@ export class EntitlementsManager {
...
+   */
+  getExperimentConfigFlags() {
+    return this.getArticle().then((article) => {
+      const expConfig = article['experimentConfig'];
+      if (expConfig != null) {
+        const expFlags = expConfig['experimentFlags'];
+        if (expFlags != null) {
+          return expFlags;
+        }
+        return [];
```
Comment (ChrisAntaki) [inline src/runtime/entitlements-manager.js:476]: Could you add a test that covers this line?
https://app.codecov.io/gh/subscriptions-project/swg-js/compare/2128/diff

Comment (polyakovn) [reply src/runtime/entitlements-manager.js:485]: Yes! Done.

File: src/runtime/entitlements-manager.js [hunk 2]
```diff
@@ -465,6 +470,23 @@ export class EntitlementsManager {
...
```
Comment (ChrisAntaki) [inline src/runtime/entitlements-manager.js:485]: Should the method return `[]` if `expConfig` is `null` ?

Comment (polyakovn) [reply src/runtime/entitlements-manager.js]: Ah yes, good catch! Updated & added another test

**Review by ChrisAntaki — APPROVED** [review]

File: src/runtime/entitlements-manager.js (hunk 2 above)
Comment (ChrisAntaki) [reply src/runtime/entitlements-manager.js]: Thanks!

**Pull request merged**